src/closure_externs.js

```diff
@@ -78,3 +78,168 @@ var PromiseConstructorLike;
 /** @typedef {function(new:symbol)} */
 var SymbolConstructor;
 
```

Comment:
Add some comment on what belongs here - "Types that are part of lib???.d.ts, but not in ???.externs". Fill out ??? exactly so that we can decide in the future whether we should be adding or removing types from there. Both TS and closure platform typings change, so we would need to update correspondingly here.